tests/test_cases/test_sv_interface/Makefile

```diff
@@ -0,0 +1,21 @@
+# This file is public domain, it can be freely copied without restrictions.
+# SPDX-License-Identifier: CC0-1.0
+
+TOPLEVEL_LANG ?= verilog
+VERILOG_SOURCES = $(shell pwd)/top.sv
+MODULE = test_sv_if
+TOPLEVEL = top
+
+# simulator behavior:
+# icarus sees SV interfaces as vpiModule but doesn't discover signals inside them
+# verilator sees SV interfaces as vpiModule and discovers signals inside them
+# commercial simulators all see SV interfaces as vpiInterface and discover signals inside them
+
+
+ifeq ($(filter questa xcelium vcs riviera verilator icarus,$(shell echo $(SIM) | tr A-Z a-z)),)
```

Comment:
This test is running when `TOPLEVEL_LANG=vhdl` is set and that is what is causing the Questa VHDL test to fail. Please guard against that.

Reply:
@ktbarrett yep, I did...Think it is ok now right? I can't view the build logs on azure...